src/test/java/com/commercetools/pingpong/model/MessageTest.java

```diff
@@ -20,4 +21,11 @@ public void shouldReturnGivenActionType() {
         assertThat("Action Type must be 1", message.getActionType(), is(1));
     }
 
+    @Test
+    public void checkAttributeType() {
```

Comment:
Hmm what's the point of this test?

Reply:
actionTypes are always integer and the buttonIDs are strings but currently they are both numbers... I wrote this test to remember that the buttonID should stay as a string and not be simplified into a integer because the IDs will maybe changed into non number values
I hope that makes sense for you 😅

Reply:
I see, we should then test it differently. You can test that the button id is a string and the action type an integer 😉

However, those values/fields are already typed by Java, so there is no need to test this particular case. Does it make sense to you?

Reply:
Hmm I think I understand.. 
was more a little reminder for me